src/templates/Footer/index.js

```diff
@@ -31,24 +31,25 @@ function Footer(props) {
   const {
     className,
     phoneNumber,
-    text,
+    emailAddress,
```

Comment:
Sorry Petr, `onClickChat` I was mistaken ignore what I said about that.

`mailto` spam has pretty common for many years. There are tons of email obfuscation scripts out there because of it. If we haven't had a problem that's good. It must mean our spam filter is very good, it's not being crawled or it's not big enough a problem for people to complain about.